packages/server/src/handlers/OptionsHandler.ts

```diff
@@ -6,7 +6,13 @@ import type http from 'node:http'
 // A successful response indicated by the 204 No Content status MUST contain
 // the Tus-Version header. It MAY include the Tus-Extension and Tus-Max-Size headers.
 export class OptionsHandler extends BaseHandler {
-  async send(_: http.IncomingMessage, res: http.ServerResponse) {
+  async send(req: http.IncomingMessage, res: http.ServerResponse) {
+    const maxSize = await this.getConfiguredMaxSize(req, '')
```

Comment:
Please add documentation for the end users, that `options.maxSize` might be called with an empty ID. Or maybe it should not be an empty string but rather explicitly `null`.

Reply:
Done